source/FunkinLua.hx

```diff
@@ -43,8 +43,8 @@ import Discord;
 using StringTools;
 
 class FunkinLua {
-	public static var Function_Stop = 1;
-	public static var Function_Continue = 0;
+	public static var Function_Stop =''Function_Stop'' ;
+	public static var Function_Continue = ''Function_Continue'';
```

Comment:
what

Reply:
-- yeah literally all this will do is create a minor performance drop for something that benefits nobody